addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -97,6 +98,7 @@ func (r *AntreaConfigReconciler) SetupWithManager(ctx context.Context, mgr ctrl.
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&cniv1alpha1.AntreaConfig{}).
 		WithOptions(options).
+		WithEventFilter(predicates.ConfigOfKindWithoutAnnotation(constants.TKGAnnotationTemplateConfig, constants.AntreaConfigKind, r.Log)).
```

Comment:
**thought**: To be extra safe can we limit this to only system namespace (i.e. tkg-system or vmware-system-tkg) . I think system namespace is configured using https://github.com/vmware-tanzu/tanzu-framework/blob/main/addons/main.go#L119 ?

@vijaykatam for any input on whether namespace filtering is required.

Reply:
+1 to filtering on system namespace

Reply:
Discussed with Vijay, he also agrees to limit the predicate to only system namespace. Will update it

Reply:
Sounds good. Namespace filter has been added.